src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy

```diff
@@ -230,7 +273,10 @@ class UnityPlugin implements Plugin<Project> {
             })
             t.coverageOptions.convention(extension.enableTestCodeCoverage.map { it ? "generateAdditionalMetrics" : null })
             t.debugCodeOptimization.convention(extension.enableTestCodeCoverage) //needed from 2020.1 and on for coverage
-            t.testFilter.convention(UnityPluginConventions.testFilter.getStringValueProvider(project))
+
+            // Command line conventions
+            //t.testFilter.convention(UnityPluginConventions.testFilter.getStringValueProvider(project))
+
```

Comment:
Why is this gone here?

Reply:
Since the testFilter convention is being set by the auto-mapping procedure now.